### ISSUE TITLE
Vanilla Nether Biomes can't be located after a Relog

### ISSUE THREAD

**Issue by SHXRKIEgg** [Someone-Else-Was-Taken/BetterNether-Reforged#53]
Trying to figure out an issue with my Better Minecraft [FORGE] Modpack.
Similar to these issues, [#41](https://github.com/Someone-Else-Was-Taken/BetterNether-Reforged/issues/41) [#42](https://github.com/Someone-Else-Was-Taken/BetterNether-Reforged/issues/42), I have found that with Better Nether installed with BYG (Removing BetterNether this issue doesn't occur as its the issue) that after you create a world and enter the nether and do /locatebiome you can locate the vanilla biome and after I save and quit and launch the game again and jump back into my game, you cant /locatebiome vanilla nether biomes anymore expect souls and valley. The initial /locatebiome works when you first made the world and joined but after the relog some reason for example the Warped Forest that was located was replaced with a Better Nether Biome. After that I cant locate Vanilla Biomes anymore no matter how far I teleported. Basically all the Vanilla Nether Biomes are getting overridden after a restart and stop spawning.

**Comment by SHXRKIEgg** [comment]
/locatebiome after Creating a World - ![1](https://user-images.githubusercontent.com/79353591/147931946-e86579cb-8ff6-4fac-aea2-2570655dc214.png)
/locatebiome after a Relog ![3](https://user-images.githubusercontent.com/79353591/147931950-4b582dbc-42c7-451a-918e-7a219fa9f754.png)
These Crimson Forest you see is from BYG

**Comment by lSherozl** [comment]
This was for me on a new world I was able to find the crimson forests but they were 40 000 blocks away, THE WARPED FOREST, apparently does not seem to spawn, I checked and tried to locate it within a distance of 100000 to -100000....From what I deduced its def a problem with better nether...https://i.imgur.com/rYVQler.png......Im playing with both BYG and BOP with bop nether set to false....don't think its an issue with either of those biome mods, maybe byg....Main issue seems to be with the better nether generation

**Comment by lSherozl** [comment]
The biomes are still there.....This is the solution, get your world seed, and create a new world with the same seed.....In the new world go to the nether, you should be able to locate the biomes with something like a nature compass, or the /locate command....Mark down the biome locations, and in your old world, even tho the /locate command does not work....you will still be able to know where the biomes are located because the nether will generate the exact same...As you can see I found them on the new world with same speed. The issue is that after u log off once, you cant use /locate again for those vanilla biomes....Here is proof the temporary solution works: https://i.imgur.com/b6JEug5.png

**Comment by lSherozl** [comment]
Yup problem is still there, its not just an issue of /locate, its that in new chunks these biomes will not spawn entirely

**Comment by SHXRKIEgg** [comment]
Update this only happens on Singleplayer
